lua/textcase/shared/utils.lua

```diff
@@ -243,6 +243,21 @@ function utils.trim_str(str, _trimmable_chars)
   return trim_info, trimmed_str
 end
 
+function utils.get_current_word_info()
```

Comment:
I think it would be good to have this method documented on a high level what it does.

Reply:
Good suggestion. Done
https://github.com/johmsalas/text-case.nvim/blob/97b8fc077080d109114f409d25bc0291b4375817/lua/textcase/shared/utils.lua#L246C6-L246C6